### ISSUE TITLE
compilation error

### ISSUE THREAD

**Issue by lifeofguenter** [ip1981/bbprb#4]
```bash
$ mvn install
...

[INFO] Fork Value is true
     [java] Warnings generated: 3
[INFO] Done FindBugs Analysis....
[INFO]
[INFO] <<< findbugs-maven-plugin:3.0.3:check (findbugs) < :findbugs @ bbprb <<<
[INFO]
[INFO]
[INFO] --- findbugs-maven-plugin:3.0.3:check (findbugs) @ bbprb ---
[INFO] BugInstance size is 3
[INFO] Error size is 0
[INFO] Total bugs: 3
[INFO] instanceof will always return true for all non-null values in org.jenkinsci.plugins.bbprb.BitbucketBuildListener.extractTrigger(AbstractBuild), since all org.jenkinsci.plugins.bbprb.BitbucketBuildTrigger are instances of org.jenkinsci.plugins.bbprb.BitbucketBuildTrigger [org.jenkinsci.plugins.bbprb.BitbucketBuildListener] At BitbucketBuildListener.java:[line 60] BC_VACUOUS_INSTANCEOF
[INFO] Possible null pointer dereference in org.jenkinsci.plugins.bbprb.BitbucketBuildListener.onStarted(AbstractBuild, TaskListener) due to return value of called method [org.jenkinsci.plugins.bbprb.BitbucketBuildListener, org.jenkinsci.plugins.bbprb.BitbucketBuildListener] Dereferenced at BitbucketBuildListener.java:[line 35]Known null at BitbucketBuildListener.java:[line 35] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[INFO] Possible null pointer dereference in org.jenkinsci.plugins.bbprb.BitbucketHookReceiver.getBitbucketTriggers() due to return value of called method [org.jenkinsci.plugins.bbprb.BitbucketHookReceiver, org.jenkinsci.plugins.bbprb.BitbucketHookReceiver] Dereferenced at BitbucketHookReceiver.java:[line 106]Known null at BitbucketHookReceiver.java:[line 106] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
[INFO]


To see bug detail using the Findbugs GUI, use the following command "mvn findbugs:gui"



[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 24:50 min
[INFO] Finished at: 2018-01-18T10:59:34+02:00
[INFO] Final Memory: 65M/329M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:findbugs-maven-plugin:3.0.3:check (findbugs) on project bbprb: failed with 3 bugs and 0 errors  -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by ip1981** [comment]
Thanks a lot. I fixed some errors in 6994abc807bf53c335c7c61669ab12832c55d05e and in f4588208039a735750560eac922471b23804c32d. Other (like with `getInstance()`) are not trivial. I can (should) replace it with simple `get()`, but need to update dependencies. Unfortunately, in one way or another new dependencies will not resolve and I can't dig deeper for now.

**Comment by ip1981** [comment]
Worked around in https://github.com/ip1981/bbprb/commit/d179d635ab8715ce42f80ca6b878575c4afee4b3
